vendor.mod

```diff
@@ -94,7 +94,7 @@ require (
 	github.com/tonistiigi/fsutil v0.0.0-20240424095704-91a3fc46842c
 	github.com/tonistiigi/go-archvariant v1.0.0
 	github.com/vbatts/tar-split v0.11.5
-	github.com/vishvananda/netlink v1.3.0
+	github.com/vishvananda/netlink v1.3.1-0.20240922070040-084abd93d350
```

Comment:
Did we have any pending PRs still in nettlink?

Otherwise I want to give it a try as well to see if they're willing to tag a new release.

Reply:
Nothing pending. So, yes, it's probably worth asking.

Reply:
I left a comment on your last merged PR 🤞

Reply:
No response yet ... shall we merge this to get tests running with it, and before it rots - then update the release tag when one's available?